.devcontainer/Dockerfile

```diff
@@ -98,22 +98,53 @@ RUN git clone https://github.com/libuvc/libuvc.git &&\
     cd ../.. && rm -r libuvc*
 
 
-RUN mkdir -p /opt/lcas/extensions 
-WORKDIR /opt/lcas/extensions 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y 
-RUN git clone https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds.git
-WORKDIR /opt/lcas/extensions/zenoh-plugin-ros2dds
-RUN bash -c "source '$HOME/.cargo/env'; cargo build --release -p zenoh-bridge-ros2dds"
-RUN install target/release/zenoh-bridge-ros2dds /usr/local/bin/
-WORKDIR /
+# RUN mkdir -p /opt/lcas/extensions 
+# WORKDIR /opt/lcas/extensions 
+# RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y 
+# RUN git clone https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds.git
+# WORKDIR /opt/lcas/extensions/zenoh-plugin-ros2dds
+# RUN bash -c "source '$HOME/.cargo/env'; cargo build --release -p zenoh-bridge-ros2dds"
+# RUN install target/release/zenoh-bridge-ros2dds /usr/local/bin/
+# WORKDIR /
```

Comment:
do we want to remove this to clean up the code?

Reply:
good point